sailfishusermanager.cpp

```diff
@@ -348,3 +349,65 @@ bool SailfishUserManager::hasAccessRights(uint uid_to_modify) {
 
     return true;
 }
+
+void SailfishUserManager::userSwitch(uint uid)
+{
+    if (!hasAccessRights(SAILFISH_UNDEFINED_UID)) {
+        auto message = QStringLiteral("Access denied");
```

Comment:
Hmm... What is `auto` in this case, any ideas? I suggest to avoid turning C++ code into a riddle.

Reply:
QString.

Reply:
Works, gives no compiler warnings and this isn't called often so no performance hit.

Reply:
ok, riddle solved. but why does it have to be a riddle? oh well, I guess some people just like riddles. feel free to ignore my ranting

Reply:
I agree that it should be clear from the line of code what the return type is but I also think that here it's pretty clear, QStringLiteral is QString and no need to repeat that. You may disagree.